Build fails if make is called with -C from a different working directory

`make` from the directory VTest sources checked out to works, but `make -C ./VTest` from the parent directory fails with:

```
make: Entering directory '/path/VTest'
awk -f src/gensequences src/sequences > src/teken_state.h
trying python3
make \
	w \
	 DEFINES= \
	 `for s in src/*.c lib/*.c; do echo ${s%.c}.o;done`
make[1]: Entering directory '/path/VTest'
make[1]: *** No rule to make target 'w'.  Stop.
make[1]: Leaving directory '/path/VTest'
make: *** [Makefile:43: vtest] Error 2

```

The problem is introduced in https://github.com/vtest/VTest/commit/ea79895f767d52d7e36e7b9fe75101f7cc1967d3

--- a/Makefile
+++ b/Makefile
@@ -41,7 +41,6 @@ LIBS=	-L/usr/local/lib \
 
 vtest: ${DEPS} ${SRCS}
 	${MAKE} \
-		${MAKEFLAGS} \
 		 DEFINES= \
 		 `for s in $(SRCS); do echo $${s%.c}.o;done`
 
@@ -60,7 +59,6 @@ test: vtest
 
 varnishtest:	${DEPS} ${SRCS}
 	${MAKE} \
-		${MAKEFLAGS} \
 		 DEFINES="-DVTEST_WITH_VTC_VARNISH -DVTEST_WITH_VTC_LOGEXPECT" \
 		 `for s in $(SRCS); do echo $${s%.c}.o;done`
 

